src/server/mod.rs

```diff
@@ -1450,17 +1452,24 @@ impl LanguageServer for LspServer {
                     command_params.tag_values.unwrap_or_default(),
                 );
 
+                let file = self.store.get_ast_file(
```

Comment:
Why is it possible to get an invalid location? And why is it appropriate to return a hard-coded location to the start of the file instead of returning an error?

Reply:
Nice catch. Since we are always replacing the entire text with each applyEdit, then we always return the location from the file base node. Updating.